src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -1914,6 +1915,9 @@ public String getSimpleName(Element e) {
 
     private SimpleElementVisitor14<String, Void> snvisitor = null;
 
+    // If `e` is a static nested class, this method will return e's simple name
+    // preceded by `.` and an outer type; this is not how JLS defines "simple
+    // name". See "Simple Name", "Qualified Name", "Fully Qualified Name".
     private String getSimpleName0(Element e) {
```

Comment:
Maybe avoid the "simple name" term then, introducing a new name or phrase if necessary.

Reply:
That method predates this PR, so I simply clarified it.